ehrenamt-justiz-backend/README.md

```
@@ -177,4 +177,24 @@
 ```
 
-![ER Model](../docs/images/ApiDocs_Backend.PNG)
+![API-docs](../docs/images/ApiDocs_Backend.PNG)
+
+# Logging
+
+Logging is controlled via configuration file application-*.yml. logback-spring.xml is no longer used.
+
+Example:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)